src/frontend/org/voltdb/dbmonitor/js/voltdb.queryrender.js

```diff
@@ -316,6 +316,11 @@ function QueryUI(queryString, userName) {
             src += '<tr>';
             for (var k = 0; k < table.data[j].length; k++) {
                 var val = table.data[j][k];
+                if (typeof val == 'string'){
+                    if (val == '0E-12'){
```

Comment:
Is this the only hardcoded case you need to handle? what about leading zeros or any spaces?